cub/docs/block_wide.rst

```diff
@@ -0,0 +1,26 @@
+.. _block-module:
+
+Block-Wide "Collective" Primitives
+==================================================
+
+.. toctree::
+   :glob:
+   :hidden:
+   :maxdepth: 2
+
+   ${repo_docs_api_path}/*Block*
+
+CUB block-level algorithms are specialized for execution by threads in the same CUDA thread block:
+
+* :cpp:class:`cub::BlockAdjacentDifference <cub::BlockAdjacentDifference>` computes the difference between adjacent items partitioned across a CUDA thread block
+* :cpp:class:`cub::BlockDiscontinuity <cub::BlockDiscontinuity>` flags discontinuities within an ordered set of items partitioned across a CUDA thread block
+* :cpp:struct:`cub::BlockExchange <cub::BlockExchange>` rearranges data partitioned across a CUDA thread block
```

Comment:
Is there actually a difference between `:cpp:class:` and `:cpp:struct:`?

Reply:
Actually, I don't know. I tried to match the reference and definition, but not sure if it makes any difference.